setup.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# Read in config file and assign variables
+outputs=`python read_config.py`
+vars=$(echo $outputs | tr "(',)" "\n")
+vararray=($vars)
+
+subscription=${vararray[0]}
+res_grp_name=${vararray[1]}
+location=${vararray[2]}
+cluster_name=${vararray[3]}
+node_count=${vararray[4]}
+vm_size=${vararray[5]}
+
+# Install Azure-CLI
+curl -L https://aka.ms/InstallAzureCli
+
+# Install kubectl - Kubernetes-CLI
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+# Make the kubectl binary executable
+chmod +x ./kubectl
+# Move the binary into your PATH
+sudo mv ./kubectl /usr/local/bin/kubectl
+
+# Install Helm cli - fetch install script and execute it locally
+curl https://raw.githubusercontent.com/helm/helm/master/scripts/get > get_helm.sh
+chmod 700 get_helm.sh
+./get_helm.sh
```

Comment:
Again worth checking whether helm exists yet, and skipping this step if it does. To my surprise on a quick check I can't find a Ubuntu package for helm, though other environments (ie, OS X with brew) do have it at a system level.